adala/runtimes/_litellm.py

```diff
@@ -153,20 +153,32 @@ def _get_usage_dict(usage: Usage, model: str) -> Dict:
     return data
 
 
-class InstructorClientMixin:
+def resolve_litellm_model_and_provider(model_name: str, provider: str):
+    """
+    When using litellm.get_model_info() some models are accessed with their provider prefix
+    while others are not.
+
+    This helper function contains logic which resolves this for supported providers
+    """
+    if "/" in model_name:
+        model_name = "/".join(model_name.split("/")[1:])
+    provider = provider.lower()
+    if provider == "vertexai":
+        provider = "vertex_ai"
+
+    return model_name, provider
+
+
+class InstructorClientMixin(BaseModel):
+    
+    instructor_mode: str = "json_mode"
```

Comment:
could we make this `instructor_mode: instructor.Mode = instructor.Mode.JSON` to give it a better-defined type?

Reply:
It has to be de/serialized as part of the Agent json blob passed from LSE, so not sure if that would work. Can test it

Reply:
Doesn't work unfortunately